pallets/liquidity-pools/src/message.rs

```diff
@@ -659,13 +659,9 @@ impl LpMessageForwarded for Message {
 		}
 	}
 
-	fn try_wrap_forward(
-		source_domain: Domain,
-		forwarding_contract: H160,
-		message: Self,
-	) -> Result<Self, DispatchError> {
+	fn try_wrap_forward(forwarding_contract: H160, message: Self) -> Result<Self, DispatchError> {
 		Ok(Self::Forwarded {
-			source_domain: source_domain.into(),
+			source_domain: Domain::Centrifuge.into(),
```

Comment:
I would move this to the forwarder pallet. If not, this means that any message we ask for to be forwarder will have a domain of `Centrifuge` by default. But this could not be the case in all scenarios. It is only for the implementation of `pallet-liquidity-pools-forwarder`

Reply:
That was my initial approach but I couldn't think of any case where that would apply.

Reply:
`Message` should be super dumb with no business logic

Reply:
Yup, I'll revert this.